packages/ripple-nuxt-tide/lib/layouts/default.vue

```diff
@@ -48,8 +48,7 @@ import { RplAlertBase } from '@dpc-sdp/ripple-alert'
 import { RplBaseLayout } from '@dpc-sdp/ripple-layout'
 import RplSiteFooter from '@dpc-sdp/ripple-site-footer'
 import RplSiteHeader from '@dpc-sdp/ripple-site-header'
-import { isTokenExpired, getToken, clearToken, isAuthenticated } from '@dpc-sdp/ripple-nuxt-tide/modules/authenticated-content/lib/authenticate'
-import { isPreviewPath } from '@dpc-sdp/ripple-nuxt-tide/modules/authenticated-content/lib/preview'
+import { clientClearToken, isAuthenticated, isPreview, isExpired } from '@dpc-sdp/ripple-nuxt-tide/modules/authenticated-content/lib/authenticate'
```

Comment:
Do we need both isAuthenticated and isExpired? Whats the difference between these now?

Reply:
isExpired has been removed (it's not needed).